rpcs3/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ set(RPCS3_SRC
 	Input/basic_mouse_handler.cpp
 	Input/ds3_pad_handler.cpp
 	Input/ds4_pad_handler.cpp
+	Input/ds5_pad_handler.cpp
```

Comment:
```suggestion
	Input/dual_sense_pad_handler.cpp
```

Reply:
should be `Input/dualsense_pad_handler.cpp`

Reply:
hmm not sur lol 
https://github.com/Ryochan7/DS4Windows/blob/jay/DS4Windows/DS4Library/InputDevices/DualSenseDevice.cs
is file name have first meter maj like that yes but if no i prefere separe, for me is Join only if have D and S in shift